selfdrive/athena/athenad.py

```diff
@@ -278,7 +293,7 @@ def getMessage(service=None, timeout=1000):
 
 
 @dispatcher.add_method
-def getVersion() -> Dict[str, str]:
+def getVersion():
```

Comment:
why remove this?

Reply:
I removed all of the dispatcher method types when I realised the hardware ones would be Any, adding them back now though!